.github/linters/semgrep/pull-request-target-code-checkout.yaml

```diff
@@ -47,6 +47,15 @@ rules:
             ...
             $JOBNAME:
               ...
+      - pattern-not-inside: |
+          needs: [..., permission-check, ...]
```

Comment:
What are these changes? These changes seem to be in the linters directory.

Reply:
I think it is a CI fix, will rebase later